logger.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  api_mgmt_name = var.apim_suffix == "" ? "cft-api-mgmt-${var.environment}" : "cft-api-mgmt-${var.apim_suffix}"
```

Comment:
I think that the "name" local variable in init.tf is used to set up the same thing, so use that one instead.  The advantage with using "name" is that it already has code to work out the environment name, so the apim_suffix variable won't be needed.